pyregion/tests/test_region.py

```diff
@@ -1,47 +1,69 @@
 import os
-import numpy as np
 from os.path import join
 from astropy.io.fits import Header
-from ..wcs_helper import fix_lon
 from .. import open as pyregion_open
+import pytest
```

Comment:
Use `from astropy.tests.helper import pytest` instead of importing `pytest` directly.
This is standard practice in Astropy-affiliated packages I think.